### DISCUSSION TITLE
Common or Separate BasePaths for Additional Statuses

### DISCUSSION THREAD

**Discussion by eric-murray** [camaraproject/DeviceStatus#55] [Polls]
We want to add a new status "connectivity" to the Device Status API.

There are two options being considered for this:
- Adding a new path to the existing basepath
  `/device-status/v0/connectivity`
- Defining a new basepath for the new status
  `/device-connectivity-status/v0/connectivity`

There are good technical arguments in favour of each option, but none that are decisive. So we are putting it to a vote. Please read [this](https://github.com/camaraproject/DeviceStatus/blob/main/documentation/SupportingDocuments/decision-support-1-or-2-apis.md) excellent summary of the arguments and vote according to your preference.

As usual for votes, please also add a comment below stating your company name and preference to ensure transparency. You can also state how strong your preference is in case of a tied vote, or maybe we have overlooked a strong argument in favour of one or other of these options.

I'll set a deadline of 14th June for votes, but please ask for an extension if you need more time to consider.

**Comment by bigludo7** [comment]
Voted an additional path for the existing /device-status basepath for Orange;
As already stated this is a slight preference as we're fine with both approach.

**Reply by eric-murray** [reply]
@bigludo7
Did you vote? I don't see any votes for the first option yet.

**Reply by bigludo7** [reply]
Oups
Thanks @eric-murray !!
Probably forget to click on vote

**Comment by monamok** [comment]
Telefónica votes for separate basepaths.

**Comment by sachintonbridge** [comment]
Vodafone votes in favor of having separate basepaths.

**Comment by JoachimDahlgren** [comment]
From Ericsson we prefer, in this case, to have one API with multiple end-points. Thereby voting for "By defining an additional path for the existing /device-status basepath"

**Comment by akoshunyadi** [comment]
The preference of Deutsche Telekom is to have 1 API with multiple endpoints. So I voted also for "By defining an additional path for the existing /device-status basepath"

**Comment by eric-murray** [comment]
Poll now closed. Winner is Option 1.
- Adding a new path to the existing basepath
  `/device-status/v0/connectivity`
